src/power/index.d.ts

```diff
@@ -0,0 +1,32 @@
+import { Multiply, Dec, Cast } from '..';
+
+// Power two numbers: https://lodash.com/docs/4.17.15#power.
```

Comment:
I think this can be the same description as in the README.md. Also, this URL doesn't lead anywhere, how about we use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow instead?